src/node_crypto.cc

```diff
@@ -1148,6 +1157,10 @@ void SSLWrap<Base>::AddMethods(Environment* env, Local<FunctionTemplate> t) {
   env->SetProtoMethod(t, "setNPNProtocols", SetNPNProtocols);
 #endif
 
+  NODE_SET_PROTOTYPE_METHOD(t, "getALPNNegotiatedProtocol",
+                            GetALPNNegotiatedProto);
+  NODE_SET_PROTOTYPE_METHOD(t, "setALPNProtocols", SetALPNProtocols);
```

Comment:
Shouldn't this be wrapped in an `#ifdef TLSEXT_TYPE_application_layer_protocol_negotiation`?